rosdep/python.yaml

```diff
@@ -6874,6 +6874,16 @@ python3-wxgtk4.0:
   fedora: [python3-wxpython4]
   gentoo: [dev-python/wxpython]
   ubuntu: [python3-wxgtk4.0]
+python3-xmlschema-pip:
+  debian:
+    pip:
+      packages: [xmlplain]
```

Comment:
This looks like the new set of rules that you're adding, but they appear to be referencing  a different packge.